CONTRIBUTE.md

```diff
@@ -0,0 +1,31 @@
+
+## How do I make a contribution?
+
+Never made an open source contribution before? Wondering how contributions work in the in our project? Here's a quick rundown!
+
+* Find an issue that you are interested in addressing or a feature that you would like to add. Create a new issue if necessary.
+* Fork the repository associated with the issue to your local GitHub organization. This means that you will have a copy of the repository under your-GitHub-username/tego-core.
+* Clone the repository to your local machine using git clone https://github.com/github-username/tego-core.git.
+* Create a new branch for your fix using git checkout -b branch-name-here.
+* Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+* If you are new to Git or GitHub see [https://guides.github.com/](https://guides.github.com/) for tutorials and guides.
+* Push your changes to your remote repository
+* Submit a pull request to the upstream tego-core repository
+* Provide a short description of the changes in the title of the pull request, reference the issue or bug number as well.
+* Provide more details of the changes in the pull description, in particular highlight any current weaknesses or outstanding problems. If your code has made changes to Crypto or protocol classes a detailed description of the changes and justification will be required. Please try to make pull requests for features separately - avoid bundling multiple issues or features into a single pull request. This makes code review easier and more manageable.
+* A maintainer will review the pull request and provide feedback where appropriate. Please be patient, our maintainers are volunteers and only have a limited amount of time to work on the project.
+* Make changes to the pull request if the reviewing maintainer recommends them.
+
+## Where can I go for help?
+
+If you need help, you can ask questions via the issue that covers your contribution. In future we will provide some additional communication channels as we get them setup.
+
+## What does the Code of Conduct mean for me?
+
+Our Code of Conduct [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md) means that you are responsible for treating everyone on the project with respect and courtesy regardless of their identity. If you are the victim of any inappropriate behavior or comments as described in our Code of Conduct, we are here for you and will do the best to ensure that the abuser is reprimanded appropriately, per our code.
```

Comment:
Like in `CODE_OF_CONDUCT.md`, please consider wrapping lines to 80 characters for readability purposes

Reply:
Done see commit fe5d9ce